app/assets/javascripts/classes/MetricNodeVM.ts

```diff
@@ -22,15 +22,17 @@ import ko = require('knockout');
 import ns = require('naturalSort');
 
 class MetricNodeVM implements BrowseNode {
+    metricName: KnockoutObservable<string>;
     name: KnockoutObservable<string>;
     children: KnockoutObservableArray<BrowseNode>;
     subfolders: KnockoutObservableArray<BrowseNode>;
     expanded: KnockoutObservable<boolean>;
     renderAs: KnockoutObservable<string>;
     visible: KnockoutObservable<boolean>;
 
-    constructor(name: string, id: string) {
-        this.name = ko.observable(name);
+    constructor(metricName: string, displayName: string, id: string) {
+        this.metricName = ko.observable(metricName);
```

Comment:
The `name` is the name to show on the part of the tree, i.e. just the final _segment_ of the metric name; so nah, I don't think it can be done in any reasonable way using the `name` attributes, but open to ideas